### DISCUSSION TITLE
Nvchad cannot identify @tailwind

### DISCUSSION THREAD

**Discussion by Asrtodex** [NvChad/NvChad#1833] [General]
is there any solution for this. 
because in my own personal config of nvim this is not a problem. 
i love nvchad because this much faster then my config using packer

**Comment by siduck** [comment]
hi @Asrtodex  can you describe your issue?

**Reply by Asrtodex** [reply]
Im developing and app using nextjs and im installing tailwindcss and other stuff in documentation of tailwindcss when it come to adding a simple 3 line of code in global css 
@tailwind base;
@tailwind components;
@tailwind utilities;

The nvchad cannot recocnice it it showing and yelow text that telling that @tailwind is unknow command. I will show the image when im on my working station.. sory because im a new on nv chad because im using my setup build of my nvim. I would love to use nvchad because it fast

**Reply by siduck** [reply]
![image](https://user-images.githubusercontent.com/59060246/225904314-e6c98e0b-f33a-44d9-97ee-e4841475ce8f.png)

you mean these errors? nvchad has nothing to do with them. its the CSS lsp that does it

**Reply by Asrtodex** [reply]
yap that what i thing how to solve this

**Comment by Asrtodex** [comment]
it's not a problem but pain in the eyes

**Reply by siduck** [reply]
@Asrtodex there must be a way to silence it, google it!
